tests/dragonfly/seeder/script-generate.lua

```diff
@@ -18,21 +18,46 @@ local total_ops = tonumber(ARGV[6])
 local min_dev = tonumber(ARGV[7])
 local data_size = tonumber(ARGV[8])
 local collection_size = tonumber(ARGV[9])
+-- Probability of each key in key_target to be a big value
+local huge_value_percentage = tonumber(ARGV[10])
+local huge_value_size = tonumber(ARGV[11])
+local huge_value_csize = tonumber(ARGV[12])
 
 -- collect all keys belonging to this script
 -- assumes exclusive ownership
 local keys = LU_collect_keys(prefix, type)
 
-LG_funcs.init(data_size, collection_size)
+LG_funcs.init(data_size, collection_size, huge_value_percentage, huge_value_size, huge_value_csize)
 local addfunc = LG_funcs['add_' .. string.lower(type)]
 local modfunc = LG_funcs['mod_' .. string.lower(type)]
+local huge_entries = LG_funcs["get_huge_entries"]
+
+local huge_keys = 0
+
+local function huge_entry()
+    local ratio = LG_funcs.huge_value_percentage / 100
+    -- [0, 1]
+    local rand = math.random()
+    local huge_entry = (ratio > rand)
+    return huge_entry
+end
 
 local function action_add()
     local key = prefix .. tostring(key_counter)
+    local op_type = string.lower(type)
+    local is_huge = false
+    if op_type ~= "string" and op_type ~= "json" then
+      is_huge = huge_entry()
+    end
```

Comment:
Please add a comment explaining that only `string` and `json` are handled here, because other types are handled below (and where)

Reply:
We don't handle json or string here, we `just roll a dice` to decide if it should be huge value or not. There are no huge values for strings or json so that's why we skip the roll

Reply:
ok, my bad, can you please add that as a comment?

Reply:
of course!